ecl/resource_ecl_network_static_route_v2_test.go

```diff
@@ -1,69 +1,127 @@
-					resource.TestCheckResourceAttr(
-						"ecl_network_static_route_v2.static_route_1", "description", ""),
-				),
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
 			},
 		},
 	})
 }
 
-func TestAccNetworkV2StaticRoute_multiGateway(t *testing.T) {
-	if testing.Short() {
-		t.Skip("skip this test in short mode")
-	}
+func TestAccNetworkV2StaticRoute_fic(t *testing.T) {
+	var staticRoute static_routes.StaticRoute
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "ecl_network_static_route_v2.test"
 
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheckStaticRoute(t) },
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheckStaticRouteFIC(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckNetworkV2StaticRouteDestroy,
 		Steps: []resource.TestStep{
-			resource.TestStep{
-				Config:      testAccNetworkV2StaticRouteMultiGateway,
-				ExpectError: regexp.MustCompile("\"internet_gw_id\": conflicts with vpn_gw_id"),
+			{
+				Config: testAccNetworkV2StaticRouteFICConfig(rName, "create", "created"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2StaticRouteExists(resourceName, &staticRoute),
+					resource.TestCheckResourceAttr(resourceName, "name", rName+"-create"),
+					resource.TestCheckResourceAttr(resourceName, "description", "created"),
+					resource.TestCheckResourceAttrSet(resourceName, "destination"),
+					resource.TestCheckResourceAttrSet(resourceName, "fic_gw_id"),
+					resource.TestCheckResourceAttr(resourceName, "nexthop", "192.168.200.1"),
+					resource.TestCheckResourceAttr(resourceName, "service_type", "fic"),
+					resource.TestCheckResourceAttrSet(resourceName, "tenant_id"),
+					resource.TestCheckResourceAttr(resourceName, "aws_gw_id", ""),
+					resource.TestCheckResourceAttr(resourceName, "azure_gw_id", ""),
+					resource.TestCheckResourceAttr(resourceName, "gcp_gw_id", ""),
+					resource.TestCheckResourceAttr(resourceName, "interdc_gw_id", ""),
+					resource.TestCheckResourceAttr(resourceName, "internet_gw_id", ""),
+					resource.TestCheckResourceAttr(resourceName, "vpn_gw_id", ""),
+				),
 			},
-			resource.TestStep{
-				Config:      testAccNetworkV2StaticRouteMultiGateway,
-				ExpectError: regexp.MustCompile("\"vpn_gw_id\": conflicts with internet_gw_id"),
+			{
+				Config: testAccNetworkV2StaticRouteFICConfig(rName, "update", "name and description are updated"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2StaticRouteExists(resourceName, &staticRoute),
+					resource.TestCheckResourceAttr(resourceName, "name", rName+"-update"),
+					resource.TestCheckResourceAttr(resourceName, "description", "name and description are updated"),
+					resource.TestCheckResourceAttrSet(resourceName, "destination"),
+					resource.TestCheckResourceAttrSet(resourceName, "fic_gw_id"),
```

Comment:
```suggestion
					resource.TestCheckResourceAttr(resourceName, "fic_gw_id", OS_FIC_GW_ID),
```
ficの場合は明確に環境変数で渡せるようになったので、こちらの方が厳密になって良いと思います。